tests/testsuite/cargo_remove/invalid_arg/stderr.log

```diff
@@ -1,6 +1,6 @@
 error: unexpected argument '--flag' found
 
-  note: to pass '--flag' as a value, use '-- --flag'
+  tip: to pass '--flag' as a value, use '-- --flag'
```

Comment:
Just saw this. I remember rustc uses `note` instead. As an official tool, would it be better to stay consistent?